file/line_file_reader.h

```diff
@@ -41,7 +41,11 @@ class LineFileReader {
   // the line to `out`, without delimiter, or returning false on failure. You
   // must check GetStatus() to determine whether the failure was just
   // end-of-file (OK status) or an I/O error (another status).
-  bool ReadLine(std::string* out);
+  // Caller can set rate limiting priority other than the default value
+  // Env::IO_TOTAL and the internal rate will be charged at the specified
```

Comment:
".. and internal rate limiter will .."? instead of "...rate..."

Reply:
Thanks for catching this.